library/Icinga/Web/Widget/Tabextension/DashboardSettings.php

```diff
@@ -1,39 +1,43 @@
 <?php
-/* Icinga Web 2 | (c) 2014 Icinga Development Team | GPLv2+ */
+/* Icinga Web 2 | (c) 2014-2022 Icinga Development Team | GPLv2+ */
```

Comment:
We don't show a from-to, only the from, which is the year the file got introduced.